### PR TITLE
Fixed loadout card css temporarily.

### PR DESCRIPTION
**Pull request by surmiran**

### CODE REVIEW

File: resources/scss/_dashboard.scss [hunk 1]
```diff
@@ -18,44 +18,15 @@
         }
     }
 
-    .loadoutCardContainer {
```
Comment (chaseconey) [inline resources/scss/_dashboard.scss:21]: This is used on the homepage. The styles are all wonky on the homepage without it.

Screenshot_9_7_20__12_24_PM

File: resources/scss/_dashboard.scss [hunk 2]
```diff
@@ -18,44 +18,15 @@
...
     .loadoutCardContainer:hover {
-        background-color: #fc9e00;
+        background-color: #FC9E00;
 
         .weaponContainer img {
             filter: $img-filter-white;
         }
     }
 
-    .titleRow {
```
Comment (chaseconey) [inline resources/scss/_dashboard.scss:41]: Same with these as well.

**Review by chaseconey — CHANGES_REQUESTED** [review]

**Comment by chaseconey** [comment]
Deployed this to staging.

**Comment by surmiran** [comment]
put aside until we know what and how to display on the front page

**Pull request closed without merging**